app/policies/group_offer_category_policy.rb

```diff
@@ -3,16 +3,18 @@ class Scope < ApplicationScope
     def resolve
       if superadmin?
         all
+      elsif department_manager?
```

Comment:
does it make sense to add this resolve all like this since they both have the same policy or would it make sense to remove this resolve method and then adjust de controller?

Reply:
I don't think we need resolve anymore.

Reply:
Yes I think we can probably remove the scope, only superadmins and department managers can access the `index` action so there's no need to restrict the scope further.

An exception would be if the group offer categories are listed in other controllers/views, but that doesn't seem to be the case here.